map.go

```diff
@@ -573,6 +628,8 @@ const (
 	UpdateNoExist MapUpdateFlags = 1 << (iota - 1)
 	// UpdateExist updates an existing element.
 	UpdateExist
+	// Allows user space to update/lookup map elements under bpf_spin_lock.
+	FLock
```

Comment:
```suggestion
	UpdateLock
```

Reply:
Addressed in 623ff45f69ae33eecd66282db41702eafba6bf66